### PR TITLE
feat(tooltip): new tooltip based on @reach-ui/tooltip

### PR DESCRIPTION
**Pull request by jigsawye**
## Summary

移除一堆 API，只保留最基本的 content
效能上應該會更好

## Test plan

### CODE REVIEW

**Comment by tw0517tw** [comment]
感覺時間延遲有點久 可是好像不能改

File: website/docs/lab/Tooltip.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  <Button>Hover me</Button>
+</Lab.Tooltip>
+```
+
+## API
+
+
+| Property  | Description                                      | Type        | Default |
+|-----------|--------------------------------------------------|-------------|---------|
+| `content` | A string or react component inside this tooltip. | `ReactNode` |         |
```
Comment (kpman) [inline website/docs/lab/Tooltip.md:32]: 這 markdown 最下面怎麼沒有多一行空白行，一般來說好像都會有？

File: website/src/theme/ReactLiveScope/index.js [hunk 2]
```diff
@@ -15,16 +15,16 @@ import * as Yup from 'yup';
...
 import * as TailorUIFormik from '../../../../src/formik';
-import * as TailorUILab from '../../../../src/lab';
 
 // Add react-live imports you need here
 const ReactLiveScope = {
   React,
   ...React,
   ...TailorUI,
-  ...TailorUILab,
+  Lab,
```
Comment (kpman) [inline website/src/theme/ReactLiveScope/index.js]: 這個如果不用 spread 要不要放在跟 React 一起？

**Comment by kpman** [comment]
這個 hover 出現的速度可以調整嗎？好像用起來比之前的慢

## now

CleanShot 2020-10-13 at 16 01 00


### lab tooltip

CleanShot 2020-10-13 at 16 00 26

**Comment by jigsawye** [comment]
@kpman 
https://github.com/reach/reach-ui/blob/51e163dcfc2b3654e5344b719398daa3775cab63/packages/tooltip/src/index.tsx#L69-L70

timeout 已死，有事燒紙

**Comment by kpman** [comment]
太難過了吧 這時間竟然寫死的

**Comment by kpman** [comment]
其他 LGTM

**Comment by jigsawye** [comment]
有人發 issue 問可否改 timer，作者有回應

> We may consider this at some point but we're going to hold for the time being. Recommendations from W3 on specific tooltip behavior are still under consideration, and I suspect there may be spec around the timer details when they eventually land on something. I'd hate to introduce an API only to be forced to take it away.

https://github.com/reach/reach-ui/issues/411#issuecomment-573970811

他還推薦了一個很猛的東西來修改 package
https://www.npmjs.com/package/patch-package

**Review by kpman — APPROVED** [review]
LGTM

**Comment by kpman** [comment]
https://inclusive-components.design/tooltips-toggletips/ 找到這個，好像可以參考

**Pull request merged**